### ISSUE TITLE
Error happned from the conbination of railties-5.0.2 and minitest 5.10.2

### ISSUE THREAD

**Issue by junaruga** [jonleighton/spring-watcher-listen#19]
Look at this Travis result of the `spring-watcher-listen`.

https://travis-ci.org/jonleighton/spring-watcher-listen/builds/231076403

```
/home/travis/build/jonleighton/spring-watcher-listen/test/apps/gems/2.4.1/gems/railties-5.0.2/lib/rails/test_unit/minitest_plugin.rb:9:in `aggregated_results': wrong number of arguments (given 1, expected 0) (ArgumentError)
```

The problem is

railties 5.0.2 (`railties-5.0.2/lib/rails/test_unit/minitest_plugin.rb:9`) is overwriting `aggregated_results` without argument. 

```
module Minitest
  class SuppressedSummaryReporter < SummaryReporter
    # Disable extra failure output after a run if output is inline.
    def aggregated_results
      super unless options[:output_inline]
    end
  end
  ...
```

But latest version minitest 5.10.2 that has been released at May 09 2017 (2 days ago), changed `aggregated_results` to set 1 argument.
https://github.com/seattlerb/minitest/commit/c6ba2afd90473b76d289562edd24f7d7ca8484f9

So, right now we should not use minitest 5.10.2.

**Comment by junaruga** [comment]
The issue was fixed, but the fixed version is not released.
https://github.com/rails/rails/pull/29022

**Comment by junaruga** [comment]
It is this commit: https://github.com/rails/rails/commit/a6dba93

**Comment by junaruga** [comment]
It looks that tomorrow fixed version `rails` will be released.
https://github.com/rails/rails/issues/29038

**Comment by junaruga** [comment]
The rails fixed version 5.0.3 and 5.1.1 were released.
